content/en/cloudcraft/getting-started/datadog-integration.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Datadog Integration
+---
+
+## Overview
+
+The integration between Datadog and Cloudcraft provides users with a streamlined workflow for monitoring and visualizing their cloud infrastructure.
+
+By leveraging Datadog’s powerful monitoring platform, users can log in to Cloudcraft with their Datadog account, move seamlessly from any resource in Cloudcraft to the most relevant views in Datadog, and automatically synchronize cloud accounts that have already been configured in Datadog for use in Cloudcraft.
+
+## Datadog Single Sign-On (SSO)
+
+Cloudcraft allows users to sign up and log in using their Datadog account. This integration offers a unified experience, linking your Datadog monitoring data seamlessly with your Cloudcraft architecture diagrams.
+
+### Signing up with Datadog SSO
+
+To get started, choose the **Sign up with Datadog** option during the Cloudcraft sign-up process. After signing up, you can log in to Cloudcraft using your Datadog credentials. This not only simplifies the login process but also enables integration between the two platforms.
+
+By using Datadog SSO, you automatically gain access to:
+
+- **Cross-platform functionality**: Move effortlessly between Cloudcraft and Datadog to analyze your cloud infrastructure and its performance.
+- **Automated cloud account synchronization**: Cloud accounts configured in Datadog sync automatically with Cloudcraft, giving you a complete view of your infrastructure in both platforms.
+
+### Enabling Datadog SSO for existing accounts
+
+If you originally signed up with a different login method, for example, Google SSO or a standard username and password, you won’t have access to the full set of Datadog integration features. To switch to Datadog SSO, [contact the Cloudcraft support team][1] and they will assist you in converting your account.
+
+## Sync cloud accounts
+
+<div class="alert alert-info">This feature supports only Amazon Web Services (AWS) accounts. Syncing with Azure or other cloud providers is not available at this time.</div>
+
+The integration between Cloudcraft and Datadog streamlines cloud account management, allowing your cloud accounts to sync automatically between the two platforms. After adding your cloud account to Datadog, it is automatically synced with Cloudcraft, so there's no need to add the account separately in Cloudcraft.
```

Comment:
```suggestion
The integration between Cloudcraft and Datadog streamlines cloud account management, allowing your cloud accounts to sync between the two platforms. When you add your cloud account to Datadog, the cloud account is automatically added to and synced with Cloudcraft, so there's no need to add the account separately in Cloudcraft.
```
Just trying to avoid repetition / add clarity.